apps/api/src/.env.development

```diff
@@ -16,3 +16,8 @@ REDIS_DB_INDEX=2
 
 NEW_RELIC_ENABLED=true
 NEW_RELIC_APP_NAME="[DEV] - api"
+
+VERCEL_CLIENT_ID=
+VERCEL_CLIENT_SECRET=
+VERCEL_REDIRECT_URI=
+VERCEL_BASE_URL=https://api.vercel.com
```

Comment:
These new environment variables should also be added to `apps/api/src/config/env-validator.ts` and to the dockerfiles & env.example found in the docker folder